docs/streams/stream-tasks/view-streams.md

```diff
@@ -20,7 +20,118 @@ View existing streams.
 
 1. To see more details and stream statistics, click a stream name.
 
-When you view a stream in the console, you will not see a backlog of messages. This screen does not persist, which means that when it is open, you will see new messages as they are sent. If you close the screen, any messages displayed will not be visible the next time that you open it.
+</TabItem>
+<TabItem value="py" label="Python SDK">
+
+View a list of existing streams.
+
+```python
+from c8 import C8Client
+
+""" For Python SDK, you can omit https:// part of the URL """
+BASE_URL = "play.paas.macrometa.io/"
+
+""" Connect to GDN """
+client = C8Client(
+    protocol='https',
+    host=BASE_URL,
+    port=443,
+    apikey='<your API key>', # Enter your API key
+    geofabric="<your-fabric>" # Enter your fabric name
+)
+
+streams = client.get_streams()
+
+print("List of streams:")
+for stream in streams:
+    print(f"- Name: {stream['name']}")
+    print(f"  Topic: {stream['topic']}")
+    print(f"  Type: {stream['type']}")
+    print(f"  Status: {stream['status']}")
+```
+
+Get stats for a specific stream.
+
+```python
+import json
+from c8 import C8Client
+
+# Define variables
+URL = "play.paas.macrometa.io"
+GEO_FABRIC = "_system" # Change this to your fabric
+API_KEY = "your-api-key" # Change this to your API key
+
+# Create client instance with API KEY
+client = C8Client(protocol='https', host=URL, port=443, apikey=API_KEY, geofabric=GEO_FABRIC)
+
+# Name of the stream to get information for
+# Do not include the prefix
+stream_name = "your-stream-name"
+
+try:
+    # Get stream stats
+    stream_stats = client.get_stream_stats(stream_name)
+
+    # Print stream stats
+    print("Stream statistics for '{}'".format(stream_name))
+    print("----------------------------------------")
+    for key, value in stream_stats.items():
+        print("{}: {}".format(key, value))
+        
+except c8.exceptions.StreamNotFoundError:
```

Comment:
@dlozina I had an issue with this line, can you check it

Reply:
Any progress on this, @dlozina-macrometa ?

Reply:
@diana-macrometa this should be resolved after committing my suggestions